projects/TransFusion/configs/transfusion_nusc_pillar_L.py

```diff
@@ -0,0 +1,397 @@
+_base_ = ['../../../configs/_base_/default_runtime.py']
+custom_imports = dict(
+    imports=['projects.TransFusion.transfusion'], allow_failed_imports=False)
+point_cloud_range = [-51.2, -51.2, -5.0, 51.2, 51.2, 3.0]
+class_names = [
+    'car',
+    'truck',
+    'construction_vehicle',
+    'bus',
+    'trailer',
+    'barrier',
+    'motorcycle',
+    'bicycle',
+    'pedestrian',
+    'traffic_cone',
+]
+metainfo = dict(classes=class_names)
+voxel_size = [0.2, 0.2, 8]
+out_size_factor = 4
+evaluation = dict(interval=1)
+dataset_type = 'NuScenesDataset'
+data_root = 'data/nuscenes/'
+data_prefix = dict(
+    pts='samples/LIDAR_TOP',
+    CAM_FRONT='samples/CAM_FRONT',
+    CAM_FRONT_LEFT='samples/CAM_FRONT_LEFT',
+    CAM_FRONT_RIGHT='samples/CAM_FRONT_RIGHT',
+    CAM_BACK='samples/CAM_BACK',
+    CAM_BACK_RIGHT='samples/CAM_BACK_RIGHT',
+    CAM_BACK_LEFT='samples/CAM_BACK_LEFT',
+    sweeps='sweeps/LIDAR_TOP')
+input_modality = dict(
+    use_lidar=True,
+    use_camera=False,
+    use_radar=False,
+    use_map=False,
+    use_external=False)
+backend_args = None
+
+model = dict(
+    type='TransFusionDetector',
```

Comment:
```suggestion
    type='TransFusion',
```
Remind to modify the corresponding name.

Reply:
rename detector class name at https://github.com/open-mmlab/mmdetection3d/pull/2547/commits/5608d19879107b656c0276185801e1a46747305d